blackbox/docs/general/dql/selects.rst

```diff
@@ -1037,7 +1037,7 @@ This is useful if used in conjunction with aggregation functions::
    shadow the table columns are used.
 
    Grouping on multi-value fields doesn't work. If such a field is encountered
-   during a group by operation an error is thrown.
+   during a group by operation an error is thrown. However, grouping for an individual field from within a multi-value column will work.
```

Comment:
change to:

> You cannot group on whole multi-value columns. You can, however, group on individual keys within a multi-value column. 

please wrap this to 79 chars

link "multi-value columns" to https://crate.io/docs/crate/reference/en/latest/general/ddl/data-types.html#object (or a more appropriate link)

also. perhaps it would be good to add an example of a query that wont work and a query that will work. what do you think?

Reply:
I think `multi value` here isn't referring to columns of type object but instead referring to arrays (multiple values inside a column) or columns of type string which have a fulltext analyzer (causing them to be stored tokenized, which means there are multiple values per document. E.g. `hello world` becomes `['hello', 'world']`)

Reply:
We could probably also refer to `unnest` as a way to group on array columns (`select sum(x) from (select unnest(arra_col) from t ) tt`

Reply:
I see! thank you! this probably deserves to be clarified in the docs then as part of this PR (both in the dml doc, and here in the dql doc)

"multi value" isn't defined elsewhere in the docs that I found

Reply:
@mechanomi you are right, the lacking definition of the term "multi value" actually started the confusion for the client